[2025-07-09T05:10:20.523+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T05:10:20.599+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_call_python_file.extract_task manual__2025-07-09T05:03:18.296639+00:00 [queued]>
[2025-07-09T05:10:20.620+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_call_python_file.extract_task manual__2025-07-09T05:03:18.296639+00:00 [queued]>
[2025-07-09T05:10:20.621+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2025-07-09T05:10:20.651+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_task> on 2025-07-09 05:03:18.296639+00:00
[2025-07-09T05:10:20.667+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'bash_call_python_file', 'extract_task', 'manual__2025-07-09T05:03:18.296639+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpe_197gaz']
[2025-07-09T05:10:20.672+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask extract_task
[2025-07-09T05:10:20.673+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=284) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-09T05:10:20.674+0000] {standard_task_runner.py:63} INFO - Started process 285 to run task
[2025-07-09T05:10:20.829+0000] {task_command.py:426} INFO - Running <TaskInstance: bash_call_python_file.extract_task manual__2025-07-09T05:03:18.296639+00:00 [running]> on host fe3fd911cec1
[2025-07-09T05:10:21.037+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bash_call_python_file' AIRFLOW_CTX_TASK_ID='extract_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T05:03:18.296639+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T05:03:18.296639+00:00'
[2025-07-09T05:10:21.039+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T05:10:21.257+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-09T05:10:21.259+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T05:10:21.287+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bash_call_python_file, task_id=extract_task, run_id=manual__2025-07-09T05:03:18.296639+00:00, execution_date=20250709T050318, start_date=20250709T051020, end_date=20250709T051021
[2025-07-09T05:10:21.338+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T05:10:21.381+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-09T05:10:21.387+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
